src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -750,6 +752,28 @@ class WindowSpecificationSegment(BaseSegment):
     )
 
 
+@ansi_dialect.segment()
+class FunctionNameSegment(BaseSegment):
+    """Base part of a function name (excluding project, schema, etc.)."""
```

Comment:
Small nit - I think this docstring is out of date now. `FunctionNameSegment` explicitly allows the option to **include** project, schema etc, and this says it excludes it.

Reply:
Ah yes -- I'll fix that now.